staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -713,7 +713,7 @@ type NamedRuleWithOperations struct {
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-
+// +k8s:prerelease-lifecycle-gen:introduced=1.16
```

Comment:
can all these tags and the verify changes be in a separate PR to sweep, verify, and separately merge.  It's easy to enforce and prevents adding more exceptions.

Might ask @jpbetz to help verify the versions being added are correct in that PR.

Reply:
opened #124898

Reply:
> opened #124898

Thanks, I'm ready to approve after seeing proof of the verify script working and either Joe or stefan confirming the versions.